### PR TITLE
Typograpghy component 

### PR DESCRIPTION
**Pull request by ronaldAlhazov**
Created the Typography component and added her to button and checkbox component.

### CODE REVIEW

File: src/Components/Typography/Typography.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { TypographyProps } from "./types";
+import { StyledTypography } from "./styles";
+
+const Typography = ({ label, type, children }: TypographyProps) => {
+  return (
+    <div>
+      <StyledTypography type={type}>{children || label}</StyledTypography>
+    </div>
```
Comment (omershafrir-moveo) [inline src/Components/Typography/Typography.tsx]: can remove the wrapping `<div><div/>` and return directly the `<StyledTypography type={type}>{children || label}</StyledTypography>`

File: src/Components/Typography/styles.ts [hunk 2]
```diff
@@ -0,0 +1,144 @@
+import "@fontsource/mulish";
+import styled from "styled-components";
+import { TypographyTypes } from "./consts";
+
+export const StyledTypography = styled.div<{ type: string }>`
```
Comment (omershafrir-moveo) [inline src/Components/Typography/styles.ts]: the type should be `TypographyTypes` and not `string`, this is exactly why we use enums in the first place.

File: src/Components/Typography/styles.ts [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            font-weight: 500;
+            line-height: 20px;
+            `;
+      case TypographyTypes.BUTTON_SMALL:
+        return `  
+            font-size: 14px;
+            font-weight: normal;
+            line-height: 14px;
+            `;
+    }
```
Comment (omershafrir-moveo) [inline src/Components/Typography/styles.ts:148]: add here a `default` case , if the type is not one of the following ( in the default case just think of the best default value- and return the most general type of typography).

**Review by omershafrir-moveo — CHANGES_REQUESTED** [review]
good work , small changes!

File: src/Components/Typography/Typography.tsx (hunk 1 above)
Comment (ronaldAlhazov) [reply src/Components/Typography/Typography.tsx]: done

File: src/Components/Typography/styles.ts (hunk 2 above)
Comment (ronaldAlhazov) [reply src/Components/Typography/styles.ts]: done

File: src/Components/Typography/styles.ts (hunk 3 above)
Comment (ronaldAlhazov) [reply src/Components/Typography/styles.ts:148]: done

**Review by omershafrir-moveo — APPROVED** [review]
Nice!

**Pull request merged**